charts/bindplane/templates/transform-agent-service.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: bindplane-transform-agent
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: bindplane-transform-agent
```

Comment:
I think this could be
```yaml
app.kubernetes.io/name: {{ include "bindplane.name" . }}-transform-agent
```

This way, it will match the name of the bindplane service, which is not always going to be "bindplane".

Reply:
I think this should apply to anywhere that we have `bindplane-transform-agent`, including the name for this service and the name for the deployment.